src/bank/two/AccountException.java

```diff
@@ -0,0 +1,7 @@
+package bank.two;
+
+class AccountException extends Exception {
+    public AccountException(String msg){
+        super(msg);
+    }
```

Comment:
Ok, ale nie widzę sensu w tworzeniu klasy identycznej do Exception tylko po to by nadać jej inną nazwę. Po prostu zostawiłbym Exception. ;)